dotcom-rendering/src/lib/signInAfterCheckOutText.ts

```diff
@@ -0,0 +1,22 @@
+// HEADER TEXT
+export const SUBSCRIPTION_HEADER = 'Thank you for subscribing';
```

Comment:
If you use JSDoc you’ll get nice IDE integration

```suggestion
/** Header text */
export const SUBSCRIPTION_HEADER = 'Thank you for subscribing';
```

Reply:
Oh nice, what's JSDoc?

Reply:
Oh, literally what you have typed out. Got it 😅